### ISSUE TITLE
Subsequental call to script https://apis.google.com/js/client.js does not create iframe

### ISSUE THREAD

**Issue by w00lf** [google/google-api-javascript-client#492]
Hi there, we are using cdn version on gapi: https://apis.google.com/js/client.js, its embed in our sites body during the render:
```html
<script src="https://apis.google.com/js/client.js?onload=someCallback"></script>
```
We are using a javascript/ruby library to speed up sites rendering time - Turbolinks. This library takes the body of the site and replaces it without rerendering the whole page, this speeds up page load. The problem is when page`s body is replaced the call to script https://apis.google.com/js/client.js does not recreate Google`s sign in iframes which client need in order to get access to the Google services if he is not logined into his account. How can we force this library to re-render these iframes? The brute force workaround is to set window variables that script using to null and then call the library,  that works for now, but i dont think tis the most cleanest way.

Before rerender of body:
![image](https://user-images.githubusercontent.com/1158036/50423832-b43e1080-086b-11e9-83ed-50670672c55f.png)
After rerender:
![image](https://user-images.githubusercontent.com/1158036/50423836-c4ee8680-086b-11e9-983f-665b44776e1b.png)

**Comment by sergentj** [comment]
Hi,
Sorry we did not respond to this in a timely fashion.
I am closing under the assumption that this is not relevant any more >2 years later.
For what it's worth, loading gapi twice is unsupported and likely to cause bugs like this in the future.

**Comment by etyp** [comment]
@sergentj Maybe it's best for me to open another issue as a question, but is there any reason that calling `gapi.client.init` loads an iframe to initialize the client?

We have a partner who we're building an integration with who has concerns over having `content.googleapis.com` in a domain allow list because it is very broad. The iFrame / proxy is coming from `content.googleapis.com`. Ideally, we'd expect a `gapi.client.init` to load libraries from their respective subdomain. For example, loading the calendar API like:

```
gapi.client.init({
    discoveryDocs: [
      'https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest',
    ],
  });
```

Would load the SDK resource from `calendar.googleapis.com` rather than `content.googleapis.com`.

Worst case scenario, we can change our code to make vanilla HTTP calls to the REST API, but it's obviously more ideal to use the out-of-the-box Javascript SDK.

**TLDR: Why does gapi.client.init load an iFrame and resources from content.googleapis.com rather than from specific googleapis.com subdomains based on what's passed into the discovery docs?**
